### ISSUE TITLE
Windows 11 - Ability to use custom icon for secondary icon (tile)

### ISSUE THREAD

**Issue by Jaiganeshkumaran** [microsoft/WindowsAppSDK#1405]
# Proposal: Ability to use custom icon for secondary icon (tile)
In the previous generation of Windows, secondary tiles (now icons) can actually have a different visual displayed than the backing app's icon. However now, all of them will use the app's icon which makes it confusing and ambiguous. The only sign that this is different is the text below. Some of the apps used the live tile feature to display a different icon/logo even if it wasn't really live.

## Summary

Provide a new API to set an icon that's different from the app's icon to emphasis that it's a secondary icon.

## Rationale

* More flexibility for developers
* Richer user experience
* Existing apps can work as they were originally intended

## Scope

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to set a custom icon/logo for a secondary icon | Must |

**Comment by sylveon** [comment]
The scope section is incomplete, it's just repeating what was already said. More could be added, like "this API will allow developers to dynamically update the icon of secondary tiles" being should, or "this API will allow an app to make itself as if it was another app" being won't (by showing the app's original icon as overlay and forcing the app name in parenthesis in the description).

**Comment by Jaiganeshkumaran** [comment]
> this API will allow an app to make itself as if it was another app" being won't (by showing the app's original icon as overlay and forcing the app name in parenthesis in the description).

I don't think there's anyway to prevent that since you could put any icon and text in a secondary tile.

**Comment by sylveon** [comment]
> by showing the app's original icon as overlay and forcing the app name in parenthesis in the description

**Comment by mshaikhcool** [comment]
Since when Tiles had become a thing in windows 11 ? 🤔

**Comment by Jaiganeshkumaran** [comment]
> Since when Tiles had become a thing in windows 11 ? 🤔

Windows 11 doesn't have live tiles however if you pin a secondary tile in your app, it gets migrated as a icon on the Start menu.

**Comment by scott-stoltz** [comment]
I agree with this. I have an app that allows users to pin tiles for various things and customize the tile appearance. This of course only works in Windows 10. Since Windows 11 doesn't have tiles anymore, the secondary tile feature just adds a new start menu item with custom text, but uses the same icon of my app. So, if a user of my app pins 5 links to the start menu, they all have my app's icon. This is very confusing. It would be great if we could at least specify a custom icon for secondary item that's pinned.

**Comment by scott-stoltz** [comment]
Just saw this: https://www.theregister.com/2023/03/20/microsoft_windows_11_defaults/
